homeassistant/components/light/plum_lightpad.py

```diff
@@ -0,0 +1,251 @@
+        return self._load.llid
+
+    @property
+    def should_poll(self):
+        """No polling needed."""
+        return False
+
+    @property
+    def name(self):
+        """Return the name of the switch if any."""
+        return self._load.name
+
+    @property
+    def brightness(self) -> int:
+        """Return the brightness of this switch between 0..255."""
+        return self._brightness
+
+    @property
+    def is_on(self) -> bool:
+        """Return true if light is on."""
+        return self._brightness > 0
+
+    @property
+    def dimmable(self):
+        """Return whether the load is configured as dimmable."""
+        return self._load.dimmable
+
+    @property
+    def device_state_attributes(self):
+        """Additional State attributes."""
+        return {
+            'llid': self.llid,
+            'brightness': self.brightness,
+            'dimmable': self.dimmable
+        }
+
+    @property
+    def supported_features(self):
+        """Flag supported features."""
+        if self.dimmable:
+            return SUPPORT_BRIGHTNESS
+        return None
+
+    def turn_on(self, **kwargs):
+        """Turn the light on."""
+        if ATTR_BRIGHTNESS in kwargs:
+            self._brightness = kwargs[ATTR_BRIGHTNESS]
+            self._load.turn_on(self._brightness)
+        else:
+            self._load.turn_on()
+
+    def turn_off(self, **kwargs):
+        """Turn the light off."""
+        self._load.turn_off()
+
+
+class GlowRing(Light):
+    """Represenation of a Plum Lightpad dimmer glow ring."""
+
+    def __init__(self, lightpad):
+        """Initialize the light."""
+        self._lightpad = lightpad
+        self._name = lightpad.friendly_name + " Glow Ring"
+        self._red = lightpad.glow_color['red']
+        self._green = lightpad.glow_color['green']
+        self._blue = lightpad.glow_color['blue']
+        self._white = lightpad.glow_color['white']
+        self._brightness = lightpad.glow_intensity * 255.0
+        lightpad.add_event_listener('configchange', self.configchange_event)
+
+    def configchange_event(self, event):
+        """Configuration change event handling."""
+        config = event['changes']
+        self._red = config['glowColor']['red']
+        self._green = config['glowColor']['green']
+        self._blue = config['glowColor']['blue']
+        self._white = config['glowColor']['white']
+        self._brightness = config['glowIntensity'] * 255.0
+        self.schedule_update_ha_state()
+
+    @property
+    def red(self):
+        """Return the Red value 0..255."""
+        return self._red
+
+    @property
+    def green(self):
+        """Return the Green value 0..255."""
+        return self._green
+
+    @property
+    def blue(self):
+        """Return the Blue value 0..255."""
+        return self._blue
+
+    @property
+    def white_value(self):
+        """White Color value from 0..255."""
+        return self._white
+
+    @property
+    def rgb_color(self):
```

Comment:
This isn't used.

Reply:
Half baked feature.  The devices supports white in addition to RGB, It's not great RGBW color.  I was thinking that it would be nice to be able to individually control each channel in addition to the color picker conversions that ignore the white value. I'd still like to be able to set it programmatically, though.

Reply:
The `rgb_color` property isn't used, so please remove it.